common/src/main/java/com/wynntils/screens/base/widgets/ItemSearchButton.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.screens.base.widgets;
+
+import com.wynntils.screens.itemfilter.ItemFilterScreen;
+import com.wynntils.utils.mc.McUtils;
+import net.minecraft.client.gui.components.Tooltip;
+import net.minecraft.client.gui.screens.Screen;
+import net.minecraft.network.chat.Component;
+
+public class ItemSearchButton extends WynntilsButton {
```

Comment:
`ItemSearchButton` is a bit too broad, I had to read the code to know what this does. I propose renaming to `ItemSearchUIButton`/`ItemFilterUIButton`.